src/System.Collections/tests/Generic/List/List.Generic.Tests.cs

```diff
@@ -52,5 +52,14 @@ protected void VerifyList(List<T> list, List<T> expectedItems)
         }
 
         #endregion
+
+        [Theory]
+        [MemberData(nameof(ValidCollectionSizes))]
+        public void CopyTo_ArgumentValidity(int count)
+        {
+            List<T> list = GenericListFactory(count);
+            Assert.Throws<ArgumentException>(() => list.CopyTo(0, new T[0], 0, count + 1));
+            Assert.Throws<ArgumentException>(() => list.CopyTo(count, new T[0], 0, 1));
+        }
```

Comment:
Yeah. Most methods for this class have their own file but I don't want to have to make a whole file for one test. If anything all of those files should be merged into this one.